packages/service-library/tests/test_archiving_utils.py

```diff
@@ -74,6 +78,32 @@ def get_dirs_and_subdris_in_path(path_to_scan: Path) -> List[Path]:
         yield temp_dir_path
 
 
+@pytest.fixture
+def file_content() -> str:
+    return "test" * 10
```

Comment:
faker?

Reply:
1. yes please ... we agreed on thisone.  Here  ``faker``'s loren provider https://faker.readthedocs.io/en/master/providers/faker.providers.lorem.html
2. really there is no need to fragment so much fixtures when they are not reused ... 
3. suggest rename ``well_known_dir``  -> ``fake_dir``

So it boils down to ...

```python
  @pytest.fixture
  def fake_dir(tmp_path: Path, faker: Faker)
      file_content = faker.text()
#      ...
  
```